thyrel-web/src/components/AppLayout.tsx

```diff
@@ -13,15 +13,13 @@ export default function AppLayout({ children }: AppLayoutProps) {
       bg={paperColor}
       display="block"
       margin="auto"
+      width={100}
```

Comment:
tu as pas tester ton composant car la tu vas avoir un width de `100px` et pas `100%` !
```suggestion
      width="100%"
```

Reply:
![Capture d’écran 2021-02-18 à 16 11 50](https://user-images.githubusercontent.com/56388157/108377152-05f04480-7204-11eb-99e6-b8565914b4e0.png)

Reply:
la page a surtout pas refresh